editor/components/block-switcher/test/multi-blocks-switcher.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import { MultiBlocksSwitcher, mapStateToProps } from '../multi-blocks-switcher';
+
+describe( 'MultiBlocksSwitcher', () => {
+	test( 'should return null when the selection is not a multi block selection.', () => {
+		const isMultiBlockSelection = false;
+		const selectedBlockUids = [
+			'an-uid',
+		];
+
+		expect( MultiBlocksSwitcher( { isMultiBlockSelection, selectedBlockUids } ) ).toBe( null );
+	} );
+
+	test( 'should return a BlockSwitcher element matching the snapshot.', () => {
+		const isMultiBlockSelection = true;
+		const selectedBlockUids = [
+			'an-uid',
+			'another-uid',
+		];
+
+		expect( MultiBlocksSwitcher( { isMultiBlockSelection, selectedBlockUids } ) ).toMatchSnapshot();
+	} );
+
+	describe( 'mapStateToProps', () => {
```

Comment:
That's interesting, it seems like a good idea to test this, too 👍

Reply:
I kind of disagree with this, I dislike the fact that we export these functions for test purpose. In general if the behavior of these functions is complex enough to deserve a test, it's complex enough to deserve a selector :)

Reply:
Would using mock stores be preferable?

Reply:
No, just testing the wrapped component instead and providing the right props. That's what we do for most components.